src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -235,16 +235,6 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, diagnosticsPool, &
       end do
       !$omp end do
 
-      !$omp do schedule(runtime) private(k)
-      do iEdge = 1, nEdges
-         do k = 1, nVertLevels
-            slopeTriadUp(k, :, iEdge) = 0.0_RKIND
-            slopeTriadDown(k, :, iEdge) = 0.0_RKIND
-            gmKappaScaling(k,iEdge) = 1.0_RKIND
```

Comment:
These variables didn't need to be reset every time step. They are computed below. I added lines to initialize them the first time in the Registry. I did a check, results of QU240 are bfb before/after this change.